versions.tf

```diff
@@ -0,0 +1,4 @@
+
+terraform {
+  required_version = ">= 0.12"
```

Comment:
You need to increase the version in the main.tf file here:
https://github.com/alex-goncharov/terraform-aws-vpc/blob/2626e910146e7c0afee8ee8338ea3d6e048059dc/main.tf#L1-L3
and remove the versions.tf file

Reply:
Thanks, fixed.